core/storage/topic/gae_models.py

```diff
@@ -166,6 +166,11 @@ class TopicModel(base_models.VersionedModel):
     # Multiply and Divide'.
     page_title_fragment_for_web = datastore_services.StringProperty(
         required=True, indexed=True)
+    # The list of skill_id that will be used from a topic in the
```

Comment:
I think in the validate() function in the domain layer for the topic, you'll need to verify that the skill IDs actually belong to the topic (i.e. they're represented somewhere in the subtopics list)?

Please also contact @lkbhitesh07 about updating the validations spreadsheet accordingly (we have a list of validations we do for all the data structures).

Reply:
Added.

Reply:
Hey!
The spreadsheet Sean mentioned, you can find that [here](https://docs.google.com/spreadsheets/d/1kNLF-UkS_Qk0igNCGwLjJXM0ByHalqli41GPvWF43PE/edit?usp=sharing) here according to the category we have several checks and the status of those checks.
If I'm not wrong here you are validating the `skill id` inside the `topic`, whether they are valid or not. If that is the case you can visit the spreadsheet and in the `topic` sheet you can look for the validation, in case it is not present there you can simply leave a suggestion following the pattern.
Please do let me know if I can help in any way.
Thanks